README.md

```diff
@@ -620,14 +623,19 @@ PGENV_INITDB_OPTS='-U postgres --locale en_US.UTF-8 --encoding UNICODE'
 
 You can edit the file and adjust parameters to your needs.
 
-In order to create a configuration file for a specific version, pass the
-version to the `write` subcommand:
+In order to create a configuration file for a specific version, it is possible
+to use the `init` or `write` commands. If no prior default configuration exists,
+the commands do the same, that is they create a from-scratch configuration file.
+If a default configuration exists, the `write` command will "clone" such configuration
+in a PostgreSQL version specific configuration file, while `init` will 
+create a blank configuration file.
```

Comment:
it will backup the current default configuration and provide a new "from scratch" one. Again, a matter of taste, I suspect you want `init` to work only once for PostgreSQL version, right?

Reply:
Yeah, and only once for the default. If the file exists, it should do nothing IMHO. It is already initialized, you know?